graph/graph_windows.go

```diff
@@ -155,6 +182,50 @@ func (graph *Graph) TarLayer(img *Image) (arch archive.Archive, err error) {
 		// We keep this functionality here so that we can still work with the VFS
 		// driver during development. VFS is not supported (and just will not work)
 		// for Windows containers.
-		return graph.driver.Diff(img.ID, img.Parent)
+		root := graph.imageRoot(img.ID)
+		mFileName := filepath.Join(root, tarDataFileName)
+		mf, err := os.Open(mFileName)
```

Comment:
I don't understand why they are split out at this level. There is no system specific stuff in here... :(